engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -542,12 +542,16 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
       case other =>
         workflowLogger.error(s"${other.tag} is not a runnable key")
         WorkflowExecutionDiff.empty.validNel
-    })
+    })} map {
+      case (key: JobKey, value: ErrorOr[WorkflowExecutionDiff]) => value.valueOr(errors => {
+        self ! JobFailedNonRetryableResponse(key, new Exception(errors.toList.mkString(System.lineSeparator)) with NoStackTrace, None)
```

Comment:
I know others frameworks sometimes do in non-test code too, but why `with NoStackTrace` here? When this does occur, is the message logged anywhere?

Reply:
My thinking was that like many of the "exceptions" in Cromwell, this is actually used for (string) message passing like in `ErrorOr`s, instead of actually being used with traditional exception semantics.

Yes, this gets logged, as single lines rather than as (not particularly helpful) stack traces. I'm not sure if it gets "metadata'd" but that's not a *regression* since prior to this PR